packages/libs/lambda-at-edge/src/default-handler.ts

```diff
@@ -590,7 +595,6 @@ const handleOriginRequest = async ({
 const handleOriginResponse = async ({
   event,
   manifest,
-  prerenderManifest,
```

Comment:
If you remove this here, you could also remove it from the call in `handler` above and the type below.